doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+| Network Scalability | Limited to a few dozen devices                 | Supports massive deployments of sensors and devices                                       |
++---------------------+------------------------------------------------+-------------------------------------------------------------------------------------------+
+| Security            | Basic security features                        | Stronger encryption (AES-128) and network hiding capabilities                             |
++---------------------+------------------------------------------------+-------------------------------------------------------------------------------------------+
+
+Key Features of DECT NR+
+************************
+
+DECT NR+ presents a compelling option for low-power embedded devices, especially in applications requiring mMTC and URLLC.
+Its focus on low power consumption, reliable data transfer, flexible network configurations, and strong security make it ideal for large sensor networks and real-time industrial applications.
+Additionally, the utilization of the license-exempt 1.9 GHz band with specific channel bandwidth and modulation schemes contributes to its deployment advantages.
+
+Building upon the improvements over DECT, NR+ offers the following key features:
+
+License-Exempt Global Band
+   DECT NR+ utilizes the globally available 1.9 GHz band, which is license-free in most regions.
+   This reduces setup costs and complexity compared to LTE, which often requires licenses.
+
+Supported Channels
+   DECT NR+ operates within the 1.9 GHz band, using a specific channel bandwidth of 125 kHz.
+   This allows for denser network deployments compared to wider-bandwidth technologies.
+
+Modulation Schemes
+   DECT NR+ supports various modulation schemes like gaussian frequency-shift keying (GFSK) and differential quadrature phase shift keying (DQPSK), which allow selection based on the desired balance between data rate, range, and robustness in different operating conditions.
+
+Built-in Coexistence
+   DECT NR+ features the built-in coexistence of multiple networks in the same location, meaning that multiple DECT NR+ networks can operate in the same area without interference, enabling flexible deployment in dense environments.
+
+Flexible and Low-Latency network topologies
+   DECT NR+ supports various network configurations, like star networks or mesh networks, adapting to specific applications.
+
+High Reliability
+   DECT NR+ employs Automatic Repeat Request (ARQ) to correct data transmission errors, ensuring reliable communication for mMTC applications.
+
+Security
+   Possibility of hiding the network using AES-128 encryption and integrity protection.
+   DECT NR+ communication can be encrypted and the network itself hidden for enhanced security.
+
+High-speed
+   DECT NR+ reaches data rates up to 3.4 Mbps, depending on modulation.
```

Comment:
I believe we are indicating what 9161/9151/9131 can do with this tput figure of 3.4 Mbps? NR+ as a standard has much higher numbers of course